main.go

```diff
@@ -1,133 +1,151 @@
-package main

```

Comment:
IDK github is rendering it like this /shrug

Reply:
i forgot to format. Yours is formatted.